ansible/roles/ooni-run/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+- name: "Install dependencies"
+  apt: name={{ item }} update_cache=yes state=latest
+  with_items:
+    - 'git'
+
+# npm needs to have a home directly set otherwise it will fail to start...
+- name: "Set ooni-run user home"
+  user:
+    name: "{{ ooni_run_user }}"
+    home: "{{ ooni_run_install_path }}"
```

Comment:
I'm afraid, it'll be overridden if `dom0-bootstrap` playbook is rerun against the host.

Reply:
Is there a way to overcome this?